library/src/scripts/styles/globalStyleVars.ts

```diff
@@ -172,7 +172,7 @@ export const globalVariables = useThemeCache(() => {
         },
         alignment: {
             headings: {
-                capitalLetterRatio: 0.715, // Calibrated for Open Sans
+                capitalLetterRatio: 0.73, // Calibrated for Open Sans
```

Comment:
@slafleche Did you suggest this change? It seems a bit arbitrary and I'd like to have some idea fo what adjusting our global offset is meant to accomplish here.

Reply:
I did. There's often extra space between the line height and the actual text. The text was very slightly getting cut off (about 1 px).